### DISCUSSION TITLE
Add ability to customize fallback beatmap backgrounds

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#13201] [Ideas]
**Describe the new feature:** 
Yes, this is already implemented in the lazer, but it could be possible to specify your own image directly in the lazer, without resorting to importing a legacy skin. Perhaps you can give the opportunity to install multiple images.

**Proposal designs of the feature:** 
People without a support can be given the opportunity to put one image. With a support - several.

**Comment by frenzibyte** [comment]
It's not quite clear what you're proposing here, is it specifying a background to use as a fallback if the beatmap doesn't have a background itself?

**Comment by Moltanicaa** [comment]
Both of them. Many old maps without background and several maps have terrible background.
Now on maps without a background, a normal gradient is displayed.
